### ISSUE TITLE
integrate MCMC Monitor

### ISSUE THREAD

**Issue by magland** [flatironinstitute/stan-playground#8]
It would be great to integrate MCMC monitor in this UI. Ideally, we could view the results during sampling, but I think that's not possible for the current tinystan. I'm I correct @WardBrian ?

But even if we cannot yet do real-time monitoring with MCMC Monitor, it would be great to be able to view the sampled data at the end of the run.

Some questions:
* Do we make the MCMC Monitor code a library? Or duplicate the code? I'm actually in favor of duplicating the code, and then figuring out how to deduplicate later. This will enable rapid development.
* How should MCMC Monitor be integrated here in terms of layout / UI?
* Do we save the results of sampling in the browser local storage? On a server?

@jsoules

**Comment by WardBrian** [comment]
I would not expect to ever be able to view partial samples before the end with tinystan's architecture. There are some ways I can imagine doing it (e.g. by peeking into the partially-unitialized output buffer mid-sample), but those wouldn't play nicely with web workers

That isn't saying it won't ever be possible, I just wouldn't plan any features for that particular "what if..." at this point

**Comment by magland** [comment]
Another possibility is to somehow export the results to MCMC Monitor which would open that app in a different tab. Then we keep the two projects separate.

**Comment by WardBrian** [comment]
yes, I'd like to add a "save" button which creates a CSV with the same contents that cmdstan would provide (except for the comment header). I think that would unlock the ability to use a lot of the different downstream analysis tools in the Stan ecosystem
